src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -693,9 +693,15 @@ void Compiler::eeSetLVdone()
         eeDispVars(info.compMethodHnd, eeVarsCount, (ICorDebugInfo::NativeVarInfo*)eeVars);
     }
 #endif // DEBUG
+    if (0 == eeVarsCount && eeVars != nullptr)
```

Comment:
Nit: Our [coding conventions](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md) specify that subexpressions should be fully parenthesized and that constants should (usually) be on the RHS, like this:

```suggestion
    if ((eeVarsCount == 0) && (eeVars != nullptr))
```